tests/test_pe_reports.py

```diff
@@ -84,7 +83,7 @@ def test_reports_log_levels(level):
             assert (
                 logging.getLevelName(logging.root.getEffectiveLevel()) == level.upper()
             ), f"root logger level should be set to {level.upper()}"
-            assert return_code is None, "main() should return success"
+            assert return_code is None, "main() should return success (0)"
```

Comment:
When you made the updates in https://github.com/cisagov/pe-reports/pull/114/commits/a77ca5bc0ebd5cb2caf07ed12ae0e0871b00b3c9, you missed this:
```suggestion
            assert return_code is None, "main() should return success"
```